torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -1928,7 +1934,7 @@ std::vector<std::vector<size_t>> compute_bucket_assignment_by_size(
     // be grouped together with other gradients and gets its own bucket.
     if (!expect_sparse_gradient.empty() &&
         expect_sparse_gradient[tensor_index]) {
-      result.push_back({tensor_index});
+          result.emplace_back(std::vector<size_t>({tensor_index}), kNoSizeLimit);
```

Comment:
wrong indent?

Reply:
Actually looks like the indent was off previously and on next line, at least according to linter